packages/venia-ui/lib/components/MagentoRoute/magentoRoute.js

```diff
@@ -34,14 +35,14 @@ const MagentoRoute = () => {
     } else if (routeError === NOT_FOUND) {
         return (
             <ErrorView>
-                <h1>{MESSAGES.get(routeError)}</h1>
+                <h1>{formatMessage({ id: MESSAGES.get(routeError) })}</h1>
```

Comment:
This will work because `formatMessage` falls back to the `id` if there is no defined message in the json. That said, I think this should actually be something like:

```js
const errorMessage = formatMessage({ 
  id: routeError,
  defaultMessage: MESSAGES.get(routeError);
});

...
<h1>
  {errorMessage}
</h1>
```

And you'd define the mapping of route error within in the translation json. You'd probably need to modify the keys to be something like `magentoRoute.notFoundError`. etc.